x-pack/plugins/case/server/routes/api/cases/comments/delete_comment.test.ts

```diff
@@ -4,47 +4,62 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { kibanaResponseFactory, RequestHandler } from 'src/core/server';
+import { httpServerMock } from 'src/core/server/mocks';
+
 import {
   createMockSavedObjectsRepository,
   createRoute,
   createRouteContext,
   mockCases,
-  mockCasesErrorTriggerData,
-} from '../__fixtures__';
-import { initDeleteCommentApi } from '../delete_comment';
-import { kibanaResponseFactory, RequestHandler } from 'src/core/server';
-import { httpServerMock } from 'src/core/server/mocks';
+  mockCaseComments,
+} from '../../__fixtures__';
+import { initDeleteCommentApi } from './delete_comment';
 
 describe('DELETE comment', () => {
   let routeHandler: RequestHandler<any, any, any>;
   beforeAll(async () => {
     routeHandler = await createRoute(initDeleteCommentApi, 'delete');
   });
-  it(`deletes the comment. responds with 204`, async () => {
+  it(`deletes the comment. responds with 200`, async () => {
     const request = httpServerMock.createKibanaRequest({
-      path: '/api/cases/comments/{comment_id}',
+      path: '/api/cases/{case_id}/comments',
```

Comment:
🙄

Reply:
I talked two folks and they agree with you I will move it back to case_id and comment_id in params but I will keep it under the `/api/cases/{case_id}/comments/{comment_id}` and it will always be under the case_id/ So we do not have two manage another resource like `case/comments` with different API (post/get/put/delete/patch) until we need it.

I hope that makes sense to you too. Thank you for bringing it to my attention and make me think more about it. Good teamwork here.

Reply:
haha i literally put a single emoji, thanks for doing all the hard work ;)